### PR TITLE
follow up on 2.2 docs review

### PR DESCRIPTION
**Pull request by c-nixon**
Some followup from the previous PR

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -88,6 +104,16 @@ REPOSITORY              TAG                 IMAGE ID            CREATED
...
 
+### Building agent binary on Linux
+
+The agent requires `v1.42+` of rustc [cargo-make](https://github.com/sagiegurari/cargo-make) to build. If the proper versions of rustc and cargo are installed; then simply run the following command to build the agent:
+
+```
+cargo build --release
+```
+
+The compiled binary will be built to `./target/release/logdna-agent`.
```
Comment (jorgebay) [inline docs/README.md:115]: (Not related to this line)

Look for `DameonSet` below in line 149.

File: docs/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 [![Rustc Version 1.42+]][rustc] [Join us on LogDNA's Public Slack]
 
-The LogDNA agent is a blazingly fast, resource-efficient log collection client, that forwards logs to [LogDNA]. This version of the agent is written in [Rust] to ensure maximum performance, and when coupled with LogDNA's web application, provides a powerful log management tool for distributed systems, including [Kubernetes] clusters.
+The LogDNA agent is a fast, resource-efficient log collection client that forwards logs to [LogDNA]. This version of the agent is written in [Rust] to ensure maximum performance, and when coupled with LogDNA's web application, provides a powerful log management tool for distributed systems, including [Kubernetes] clusters.
```
Comment (jorgebay) [inline docs/README.md:5]: Should these changes be applied to the root README file as well?

File: docs/README.md [hunk 3]
```diff
@@ -20,19 +20,38 @@ The LogDNA agent is a blazingly fast, resource-efficient log collection client,
...
+by the current commit which may not be ready for general use.
+
+You MUST ensure that the current tree is checked out to the branch/tag is the
+version you intend to install, if you install a pre-release version of the
+agent your logs may not be collected.
+
+To install a specific version you should ensure you checkout the tag for that
+version before applying any YAML files from the current tree.
+
+For example to install the a particular 2.2 beta you would run
```
Comment (jorgebay) [inline docs/README.md:48]:
```suggestion
For example to install a particular 2.2 beta you would run
```

File: docs/README.md [hunk 4]
```diff
@@ -20,19 +20,38 @@ The LogDNA agent is a blazingly fast, resource-efficient log collection client,
...
+```bash
+git checkout 2.2.0-beta.10
+```
+in the repo's root directory before following the install instructions relevant
```
Comment (jorgebay) [inline docs/README.md:52]: NIT: code blocks with 3 backticks are represented as a `pre` block in html breaking the current `p` paragraph, so ideally it should be preceded and followed by paragraphs.

Also, some IDEs and markdown viewers (not the case for GH) don't like code blocks not separated by empty lines.

**Review by jorgebay — COMMENTED** [review]
Looks great, I've added some minor suggestions below.

File: docs/README.md (hunk 2 above)
Comment (tanberry) [reply docs/README.md:5]: Yes... though I didn't realize there was a readme.md in the root dir. I originally assumed this one was in root, but it is in /docs. Which isn't maybe typical?

**Review by tanberry — APPROVED** [review]

**Review by jorgebay — APPROVED** [review]

**Pull request merged**